VimeoNetworking/Sources/ResponseCache.swift

```diff
@@ -113,26 +113,26 @@ final internal class ResponseCache
     
     private class ResponseMemoryCache: Cache
     {
-        private let cache = NSCache()
+        private let cache = NSCache<AnyObject, AnyObject>()
         
-        private func setResponseDictionary(responseDictionary: VimeoClient.ResponseDictionary, forKey key: String)
+        func setResponseDictionary(responseDictionary: VimeoClient.ResponseDictionary, forKey key: String)
         {
-            self.cache.setObject(responseDictionary, forKey: key)
+            self.cache.setObject(responseDictionary as AnyObject, forKey: key as AnyObject)
         }
         
-        private func responseDictionary(forKey key: String, completion: ResponseDictionaryClosure)
+        func responseDictionary(forKey key: String, completion: @escaping ResponseDictionaryClosure)
         {
-            let responseDictionary = self.cache.objectForKey(key) as? VimeoClient.ResponseDictionary
+            let responseDictionary = self.cache.object(forKey: key as AnyObject) as? VimeoClient.ResponseDictionary
             
-            completion(responseDictionary: responseDictionary)
+            completion(responseDictionary)
         }
         
-        private func removeResponseDictionary(forKey key: String)
+        func removeResponseDictionary(forKey key: String)
         {
-            self.cache.removeObjectForKey(key)
+            self.cache.removeObject(forKey: key as AnyObject)
         }
         
-        private func removeAllResponseDictionaries()
+        func removeAllResponseDictionaries()
```

Comment:
The warning we get when these protocol methods are private is:
> Method 'removeAllResponseDictionaries()' must be as accessible as its enclosing type because it matches a requirement in protocol 'Cache'

To me that sounds like `private` should be fine since both the protocol and the class are private. But apparently also marking these methods as private is _too_ restrictive?